doc/source/whatsnew/v0.24.0.txt

```diff
@@ -159,6 +159,52 @@ This is the same behavior as ``Series.values`` for categorical data. See
 :ref:`whatsnew_0240.api_breaking.interval_values` for more.
 
 
+.. _whatsnew_0240.enhancements.duplicated_inverse:
+
+The `duplicated`-method has gained the `return_inverse` kwarg
```

Comment:
double backticks were you now use single backticks (also on the lines below)

(in rst, double backticks give code-styled text)